### PR TITLE
Add Try-Catch to the SoapAPI call and include Sapmon Version

### PR DESCRIPTION
**Pull request by hsridharan**

### CODE REVIEW

**Review by PakDLiu — CHANGES_REQUESTED** [review]
Can you write a little about the testing you have done?
Otherwise, LGTM

**Comment by hsridharan** [comment]
@PakDLiu - I tested the try-catch block running the code locally when the NW SAP System was down and I was able to see the log messages accordingly. Also Verified the Sapmon column in the LA tables.

**Review by PakDLiu — DISMISSED** [review]

File: sapmon/payload/provider/sapnetweaver.py [hunk 1]
```diff
@@ -433,6 +439,9 @@ def _actionExecuteEnqGetStatistic(self, apiName: str, filterFeatures: list, filt
 
     def generateJsonString(self) -> str:
         self.tracer.info("[%s] converting result to json string" % self.fullName)
+        if self.lastResult is not None and len(self.lastResult) != 0:
+            for result in self.lastResult:
```
Comment (persiaAziz) [inline sapmon/payload/provider/sapnetweaver.py:443]: Please follow this convention to add the payload version. We use this convention in other providers as well. https://github.com/Azure/AzureMonitorForSAPSolutions/blob/f5b31350a0ae92a936147b4a5975b6aad8d3a8c7/sapmon/payload/provider/saphana.py#L306

**Review by persiaAziz — CHANGES_REQUESTED** [review]

File: sapmon/payload/provider/sapnetweaver.py (hunk 1 above)
Comment (hsridharan) [reply sapmon/payload/provider/sapnetweaver.py:443]: @persiaAziz - Renamed the column name as recommended.

**Review by persiaAziz — APPROVED** [review]

**Review by PakDLiu — APPROVED** [review]

**Pull request merged**